src/index.js

```diff
@@ -25,37 +25,33 @@ import './images/001-ticket.svg';
 import './images/customers.svg';
 import './images/login.svg';
 import './images/android-chrome-512x512.png'
+import './images/Pending.svg'
 
-const main = $('main');
 const dateSection = $('#date')
 const userBtns = $('.user-buttons');
 const body = $('body');
 const contentSection = $('#destinations-cards');
 const welcomeBanner = $('.welcome')
 const pageBanner = $('.banner')
 let today = moment().format("YYYY/MM/DD");
-let logInBtn = $('#login-btn');
 let user = new User();
-let request = null;
 
 // Non constant DOM elements
-let customersBtn;
-let requestBtn;
-let tripsBtn;
-let submitLogin;
 let logInUsername;
 let logInPassword;
-let logOutButton;
-let destinationsBtn;
 let tripModleDataSelect;
 let requestLeaveDatePicker;
 let requestReturnDatePicker;
 let requestDestination;
 let requestTravlers;
 let requestSubmit;
-let requestForm;
 let requestTotal;
 
+const closeModle = () => {
+  event.target.closest('section').remove()
```

Comment:
This could be wrapped in a jQuery object to hit jQuery spec but obviously not necessary or more efficient.